state/mocks/evidence_pool.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery 2.9.0. DO NOT EDIT.
+// Code generated by mockery 2.7.5. DO NOT EDIT.
```

Comment:
Why is this using an _older_ version of `mockery`???

Reply:
idk, I think I just had an older version of mockery installed, good catch! Weirdly enough, I couldn't actually get mockery to work using the provided scripts and the updated version. Simply calling go generate ./... instead does however 

https://github.com/celestiaorg/celestia-core/pull/631/commits/90e4244bcdc771108acd31627cf6331c202ec083